spanish-translations.md

```diff
@@ -0,0 +1,85 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our new
+tutorials (scripts and subtitles). How did we make this possible? You might think that
+we hired a professional who takes care of translating the content. As great as human beings, 
+I'm proud to say that you would be wrong! As programmers, we love to automate processes, so, that's
+exactly what we did! We leverage the powerful translations service of *DeepL*, which is
+a translation service based on an AI algorithm. And... it's surprisingly good! But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so that some technical words or code blocks won't get translated and also keep
+the translation as accurate as possible. 
+
+## DeepL API
+
+DeepL allows you to wrap your content with XML tags to give it a special meaning, for example, 
+you can wrap a word with an "ignore" tag in case you don't want it to be translated, or, you can 
+wrap a few text lines with a "non-splitting" tag to handle them as part of the 
+same sentence (useful for not loosing context on the subtitles). For this purpose we had to 
+implement a custom Markdown parser that converts MD tags into specific XML tags, so for example,
+an inlined code block `$blocks = []` is parsed into `<dp-ignore>$blocks = []</dp-ignore>`. If you got
+curious, DeepL has an API [simulator](https://www.deepl.com/es/docs-api/simulator/) where you can see all of its features. 
+We got to use them all!
+
+By the way, this is how the API call to translate a script file looks like:
```

Comment:
```suggestion
By the way, this is what the API call to translate a script file looks like:
```

Reply:
Probably we should mention/link to the GitHub package we're using.

Also, I think we need a few more details here - we might need to show the entire class (even if shortened). Specifically, I want to show what `$this->deepl` is and also an example of what those two static methods return.